mne/io/brainvision/tests/data/test_highpass_hz.vhdr

```diff
@@ -0,0 +1,103 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
```

Comment:
See above notes on @agramfort's similar comment. To actually do the changes, you need to parse the files a bit. My hesitation there is that the tests themselves become complicated enough that they could be buggy themselves. 

And on the size front: Git will actually pack such similar, related files quite efficiently.